### DISCUSSION TITLE
example compose file - SNMP port?

### DISCUSSION THREAD

**Discussion by 80211WiGuy** [librenms/docker#372] [Q&A]
Looking over the docker-compose.yml example provided, shouldn't SNMP (UDP 162) be listening on the host somewhere?
docker/examples/compose/docker-compose.yml 

Sorry if I'm mistaken - very new to docker and systems management, just a network guy trying to setup librenms for the first time.  I'm having issues trying to add devices with SNMP (no response).

**Comment by crazy-max** [comment]
@80211WiGuy  #169
